### PR TITLE
Add kubectl serial skew jobs for 1.6/master

### PR DESCRIPTION
**Pull request by ncdc**
Add kubectl serial skew jobs for 1.6/master and make sure that the
parallel kubectl skew jobs for 1.6/master skip [Serial] tests.

Fixes https://github.com/kubernetes/kubernetes/issues/43263

@ixdy @pwittrock @liggitt PTAL

cc @ethernetdan @calebamiles @marun

### CODE REVIEW

**Comment by ncdc** [comment]
This was a lot of copying and pasting so please doublecheck for accuracy!

**Comment by ncdc** [comment]
Working on fixing the jobs config file.

**Comment by ncdc** [comment]
Let's see if third time's the charm

**Comment by ncdc** [comment]
cc @rmmh @lavalamp

File: jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+E2E_OPT=--check_version_skew=false
+GINKGO_PARALLEL=n
+GINKGO_TEST_ARGS=--ginkgo.focus=Kubectl.*\[Serial\]
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest
+JENKINS_PUBLISHED_VERSION=ci/latest-1.6
+PROJECT=k8s-gce-cvm-1-6-mstr-ctl-skew-serial
```
Comment (ixdy) [inline jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env]: the project name has to be 30 characters or fewer. I'm not sure on a good alternative name.

**Comment by ncdc** [comment]
@ixdy I only did master vs 1.6 because it wasn't until commit f4d33396124cc297c05f57054359133733f9db4c went in that it started having an effect on parallel tests. We could broaden it if we need to.

File: jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env (hunk 1 above)
Comment (ncdc) [inline jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env]: k8s-gce-cvm-1-6-m-ctl-skw-ser? /yuck

Comment (ixdy) [inline jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env]: that's probably about as good as we'll get, yeah :\

Comment (liggitt) [inline jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env]: Wow

Comment (ixdy) [inline jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env]: we have work underway to stop treating projects like pets, which will help here.

**Comment by ncdc** [comment]
@ixdy updated the project names, ptal

File: jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env [hunk 2]
```diff
@@ -0,0 +1,9 @@
+E2E_OPT=--check_version_skew=false
+GINKGO_PARALLEL=n
+GINKGO_TEST_ARGS=--ginkgo.focus=Kubectl.*\[Serial\]
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest
+JENKINS_PUBLISHED_VERSION=ci/latest-1.6
+PROJECT=k8s-gce-cvm-1-6-m-ctl-skw-srl
+KUBE_NODE_OS_DISTRIBUTION=debian
+
+KUBEKINS_TIMEOUT=120m
```
Comment (ixdy) [inline jobs/ci-kubernetes-e2e-gce-1.6-master-cvm-kubectl-skew-serial.env]: wonder if 2h will be enough.

**Comment by ixdy** [comment]
/lgtm

Creating projects now (yuck):
* k8s-gce-cvm-1-6-m-ctl-skw-srl
* k8s-gce-cvm-m-1-6-ctl-skw-srl
* k8s-gce-gci-1-6-m-ctl-skw-srl
* k8s-gce-gci-m-1-6-ctl-skw-srl
* k8s-gke-cvm-1-6-m-ctl-skw-srl
* k8s-gke-cvm-m-1-6-ctl-skw-srl
* k8s-gke-gci-1-6-m-ctl-skw-srl
* k8s-gke-gci-m-1-6-ctl-skw-srl

**Comment by ethernetdan** [comment]
@ixdy has this been deployed?

**Comment by ixdy** [comment]
Not yet. Still working on getting the GKE projects set up.

**Comment by ixdy** [comment]
GKE jobs may fail for the next few hours.

**Pull request merged**